src/tlo/scenario.py

```diff
@@ -234,6 +234,11 @@ def draw_parameters(self, draw_number, rng):
         """
         return None
 
+    def draw_name(self, draw_number) -> str:
+        """Returns the name of the draw corresponding to the given draw number. This is offered for convenience so that
+        the logfile contain a 'user-friendly' label for the draw."""
+        return ""
```

Comment:
Can the default be this instead?

```suggestion
        return str(draw_number)
```